### PR TITLE
[4.0] tbody-icons with better contrast

### PR DESCRIPTION
**Pull request by chmst**
The icons unpublished and trashed have low contrast. This PR improves the contrast and removes the border from all icons. 
When the backend templated started, there were less borders in tables. Now border in the table and border around the icons seems too much. This is a matter of taste, but I suggest to make the icons better visible and the image clearer.

tbody-icons

Test needs npm run build:css

### CODE REVIEW

**Comment by brianteeman** [comment]
If they were just icons I might agree but they have functionality as well and to me just being an icon on its own does not convey that

**Comment by chmst** [comment]
With a tiny border and a tiny shadow it looks more like a klickable element. 

tbody-icons-with-border

**Comment by brianteeman** [comment]
thanks for listening - lgtm

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 1]
```diff
@@ -36,28 +36,26 @@
...
-    height: 26px;
-    font-size: 1rem;
+    width: 24px;
+    height: 24px;
+    font-size: 16px;
     line-height: 22px;
-    color: $gray-400;
-    border: 2px solid var(--border);
+    color: var(--atum-text-dark);
+    border:1px solid var(--atum-text-dark);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_icons.scss]:
```suggestion
    border: 1px solid var(--atum-text-dark);
```

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 2]
```diff
@@ -36,28 +36,26 @@
...
     border-radius: 50%;
+    box-shadow: 0px 1px 0px 0 rgba(0,0,0,.20);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_icons.scss:46]:
```suggestion
    box-shadow: 0 1px 0 0 rgba(0,0,0,.20);
```

Comment (Quy) [reply administrator/templates/atum/scss/blocks/_icons.scss:46]: `0` should not have `px`. Why the 4th value does not have `px` and the 1st and 3rd values do?

**Comment by Quy** [comment]
Featured/Unfeatured are not distinguishable.

**Comment by chmst** [comment]
Thank you, now it should be all right. 

tbody-icons

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 3]
```diff
@@ -36,28 +36,44 @@
...
   }
 
   .icon-publish {
     color: var(--success);
     border-color: var(--success);
   }
 
+  .icon-unpublish {
+    color: var(--danger);
+	border-color: var(--success);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_icons.scss]:
```suggestion
    border-color: var(--success);
```

**Comment by Quy** [comment]
Unpublished styling is different. 

Maybe I have come accustomed to the old styling, I prefer it over this PR.

26533-unpublished

**Comment by AdityaTaware** [comment]
I have tested this item :white_check_mark: successfully on 5227921db15715fdf193658399cddb05777376b2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26533">issues.joomla.org/tracker/joomla-cms/26533</a>.</sub>

**Comment by uthorat** [comment]
I have tested this item :white_check_mark: successfully on 5227921db15715fdf193658399cddb05777376b2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26533">issues.joomla.org/tracker/joomla-cms/26533</a>.</sub>

**Comment by snehaM26** [comment]
I have tested this item :white_check_mark: successfully on 5227921db15715fdf193658399cddb05777376b2<br /><br />Test successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26533">issues.joomla.org/tracker/joomla-cms/26533</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26533">issues.joomla.org/tracker/joomla-cms/26533</a>.</sub>

**Comment by Quy** [comment]
There is one suggested change.

**Comment by chmst** [comment]
Please @alikon remove RTC, I agree with @Quy - had too much to do during PBF

**Comment by chmst** [comment]
@Quy with this pr it was my inention to have better contrast for the icons. But this and the colour of buttons i a matter of taste. 
There are many UX experts around ...  they should say what is best solution. So I'll cose this PR. Thank you for you time.

**Pull request closed without merging**